### DISCUSSION TITLE
Is dual head streaming possible?

### DISCUSSION THREAD

**Discussion by markol4** [LizardByte/community#104] [Sunshine] [answered]
Hi,

I have a VM (headless, 2x HDMI dummy plugs, Windows) running with two displays and a main host (also Windows) with four displays. 
Is it possible to stream simultaneously two VM displays to my host machine? I'm using Sunshine as a streaming server and Moonlight as a client.

I was able to stream one of my VM displays at a time (it is also possible to switch interactively with Ctrl+Alt+Shift+F1/F2), but never two at the same time.

Is it even possible for Sunshine to stream two displays at the same time (maybe two instances are needed, but how to make a proper config)? Is Moonlight capable of receiving two streams from one machine (again, perhaps two instances of Moonlight)? I could not get it done so far.

I hope it is doable and someone can share some ideas.

Regards,
Marek.

**Comment by ReenigneArcher** [comment, marked as answer]
With multiple instances it might be possible, but this is not going to be an optimal experience.
